tests/store/crud/update/test_update.py

```diff
@@ -82,6 +87,31 @@ def test_update_sample_reads_illumina_fail_q30(
     assert sample.reads == 0
 
 
+def test_update_sample_reads_illumina_negative_control(
+    store_with_illumina_sequencing_data: Store, selected_novaseq_x_sample_ids: list[str]
+):
+    # GIVEN a store with Illumina Sequencing Runs and a sample id
+    sample: Sample = store_with_illumina_sequencing_data.get_sample_by_internal_id(
+        selected_novaseq_x_sample_ids[0]
+    )
+    assert sample.reads == 0
+    # GIVEN that the sample is a negative control sample
+    sample.control = ControlOptions.NEGATIVE
+    assert sample.is_negative_control is True
```

Comment:
```Suggestion

```

 think this assertion is unnecessary - we have set it the line above and we are not testing the `is_negative_control` property.